py/private/py_pex_binary.bzl

```diff
@@ -113,10 +113,18 @@ def _py_python_pex_impl(ctx):
     )
     args.add(output, format = "--output-file=%s")
 
+    # The argument list can become quite long which can throw java.io.IOException Argument list too long.
+    # Instead write the args to a file and read that file at runtime
+    args_file = ctx.actions.declare_file("args.txt")
```

Comment:
https://bazel.build/rules/lib/builtins/Args.html#use_param_file is what you want here.

Reply:
Thanks, I hadn't seen that api before. I don't write rules often.

Reply:
Let me know when you make the changes, i'll review again.